### ISSUE TITLE
Languages and translations

### ISSUE THREAD

**Issue by giuliogatto** [simevo/spid-wordpress#17]
Let's discuss the options for languages and translations

**Comment by simevo** [comment]
for comparison, spid-smart-button supports it, en and de:
https://github.com/italia/spid-smart-button/blob/master/src/data/spidI18n.js

**Comment by simevo** [comment]
This is a plugin with a definite national scope. The languages that we need to support are the three above with the possible future addition of fr (see https://github.com/italia/spid-smart-button/issues/16#issuecomment-431085405) 

I am against forcing a dependent plugin upon our users. Better use WP "native" i9n functions:
https://developer.wordpress.org/plugins/internationalization/how-to-internationalize-your-plugin/

**Comment by michaeltieso** [comment]
Agreed. This won't be an issue at all. We'll use WP default `gettext` and provide language packs via GlotPress (translate.wordpress.org) which get added automatically. I'll add in the text domain to the README shortly.

**Comment by simevo** [comment]
Perfect, in the meantime I have prepared [a PR for spid-smart-button to also support French](https://github.com/italia/spid-smart-button/pull/72).
Our objective would then be to support [it, en, de, fr].
I volunteer for the French and German translation.
